part1.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+public function postAction()
+{
+//avoid notices warnings
+    !isset($_POST['livechat_license_number']) ? $livechat_license_number = '' : $livechat_license_number =
+        $_POST['livechat_license_number'];
+    !isset($_POST['livechat_groups']) ? $livechat_groups = '0' : $livechat_groups = $_POST['livechat_groups'];
+    !isset($_POST['livechat_params']) ? $livechat_params = '' : $livechat_params = $_POST['livechat_params'];
+    $config_table = Mage::getSingleton('core/resource')->getTableName('core_config_data');
+    $read = Mage::getSingleton('core/resource')->getConnection('core_read');
+    $query = 'SELECT * FROM ' . $config_table;
+    $query .= ' WHERE scope="default" AND scope_id=0 AND path="livechat/general/license"';

```

Comment:
Its not good practice to use the database query to get and set the config value.
$store = Mage::app()->getStore(); // store info
$license= Mage::getStoreConfig('livechat/general/license', $store);